libpod/define/info.go

```diff
@@ -27,6 +27,9 @@ type HostInfo struct {
 	BuildahVersion string           `json:"buildahVersion"`
 	CgroupManager  string           `json:"cgroupManager"`
 	CGroupsVersion string           `json:"cgroupVersion"`
+	MemoryLimit    bool             `json:"memoryLimit"`
```

Comment:
I think this would be better in a subgroup,  CGroupInfo {
     CPUShares...
     MemoryLimit ...
     PidsLimit ...
}

And Alphabetized.

Reply:
@rhatdan Sure makes sense, much cleaner i'll do the required changes.

Reply:
@rhatdan This is resolved. But format might change as there is a discussion going below for `CgroupControllers []string`

Reply:
@rhatdan Made some changes in latest commit , added `CgroupControllers []string` instead. Could you please review that.